### DISCUSSION TITLE
为什么现在chatgpt模型只剩下3.5turbo，4turbo，和4turbo-version，其他的版本怎么不显示了呀

### DISCUSSION THREAD

**Discussion by dao-zero** [lobehub/lobehub#1540] [General Question] [answered]
### 💻 系统环境

Windows

### 📦 部署环境

Docker

### 🌐 浏览器

Chrome

### 🐛 问题描述

为什么现在chatgpt模型只剩下3.5turbo，4turbo，和4turbo-version，其他的版本怎么不显示了呀

### 🚦 期望结果

希望显示所有模型

### 📷 复现步骤

目前版本是0.133.3

[a comment by an automated account is not shown]

**Comment by arvinxx** [comment, marked as answer]
因为没必要。gpt-3.5-turbo 已经指向最新的 gpt-3.5-turbo-0125 ，也是16k上下文了。那么只保留这个就够了。然后 gpt-4系列也是，只要留一个 4-preview 和 vision preview 就够了。剩下的有需要完全可以自行添加 CustomModel

**Reply by DEAN-Cherry** [reply]
那为什么少了 `gpt-4` 呢？`gpt-4` 和 `gpt-4-turbo` 还是不一样的
![image](https://github.com/lobehub/lobe-chat/assets/76607677/f2e5c176-6ff9-425e-a2fe-8ba5bd7a49b0)

**Reply by arvinxx** [reply]
官方也不推荐 GPT4 了呀？都建议使用turbo了，所以我也隐藏了。

[a comment by an automated account is not shown]
